web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -230,10 +231,9 @@ public void contextInitialized(ServletContextEvent event) {
 				if (StringUtils.hasLength(appDataRuntimeProperty)) {
 					OpenmrsUtil.setApplicationDataDirectory(null);
 				}
-				
 				//ensure that we always log the runtime properties file that we are using
 				//since openmrs is just booting, the log levels are not yet set. TRUNK-4835
-				OpenmrsLoggingUtil.applyLogLevel(getClass().toString(), "INFO");
```

Comment:
Uh... If this is broken, we have a much bigger problem, since this is also what we use to set the logging levels via GP. I suggest we try to work that out rather than this.

Reply:
@ibacher would you rather we just set this to `INFO`? https://github.com/openmrs/openmrs-core/blob/master/webapp/src/main/resources/log4j2.xml#L48
 For it would fix this problem?

Reply:
I first tried this, and it worked. However, I refrained from using it since it introduces additional log lines that were not present before.

Reply:
> For it would fix this problem?

Well, my point was more that if the `applyLogLevel()` call doesn't function correctly, then changing logging via the GP will no longer work.

How we go about changing the logging level here isn't too important.

Reply:
@wikumChamith can we create a ticket for this?

Reply:
ticket: https://openmrs.atlassian.net/browse/TRUNK-6325

Reply:
Looks like this only needs a simple fix :)

New PR: https://github.com/openmrs/openmrs-core/pull/4967